metrics/serve.go

```diff
@@ -242,10 +245,22 @@ func (c *Collector) recreateMetrics(unsafeGetState readOnlyStateFn) {
 		c.nativeHistograms[id] = histogram
 	}
 
+	// Mimic some quantile objectives.
+	objectives := map[float64]float64{}
+	if c.cfg.SummaryObjectives > 0 {
+		parts := 100 / c.cfg.SummaryObjectives
```

Comment:
qq: is it valid for summary objectives to be greater than 100?

Reply:
No, they should be from 0.0 to 1.0 - you can see I am dividing by 100.0 later on. Using 100 to have cheap integer mitigation of float precision issues when substracting things later on (: